README.md

```diff
@@ -141,6 +141,20 @@ Commands:
   run
 ```
 
+## Docker Usage
+
+Using an online cluster:
+```
+docker run -it -v ~/.kube:/root/config:ro -v ~/.minikube:/root/.minikube:ro inovex/illuminatio clean run
+```
+
+Using minikube:
+Make sure to replace all absolute paths in your kubeconfig with paths that are relative to your kubeconfig:
+`/home/user/.minikube` -> `../.minikube`
+```
+docker run -it --net=host -v ~/.kube:/root/.kube:ro -v ~/.minikube:/root/.minikube:ro inovex/illuminatio illuminatio clean run
```

Comment:
Actually the main intent, at least from my side. Was to run illuminatio as a Kubernetes `Job` but this is another Task. Thanks for writing the documentation for this (and opening an issue for the describe job use case ;) )

Reply:
This PR merely addresses https://github.com/inovex/illuminatio/issues/23, I would treat https://github.com/inovex/illuminatio/issues/26 as a separate yet related feature.